lib/un_locode/locode.rb

```diff
@@ -29,9 +29,9 @@ def self.find_by_name_and_function name, function, limit = 10
     end
 
     def self.find_by_locode(locode)
-      locode = locode.split(' ')
-      includes(:country).where(city_code: locode.last)
-        .where(countries: { code: locode.first }).first
+      locode = locode.gsub(' ','').upcase
```

Comment:
Space missing after comma.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.